extractors/python_functions/gazetteer/code_snippet.md

```diff
@@ -1,10 +1,11 @@
 ```python
 # the lookup list values can also come from a knowledge base, e.g.:
 # from knowledge import my_lookup_list
+from typing import List
 
-YOUR_ATTRIBUTE = "details"
-YOUR_LABEL = "PERSON"
-LOOKUP_VALUES = ["Max"]
+YOUR_ATTRIBUTE: str = "text"
+YOUR_LABEL: str = "PERSON"
+LOOKUP_VALUES: List[str] = ["Max"]
```

Comment:
YOUR_ missing

Reply:
from knowledge base

Reply:
i think i found a good solution for the lookup list think we can use for all these instances. @leo or @divyanshukatiyar  please take a look if it works for you :)

Reply:
solved with the new approach visible here: https://github.com/code-kern-ai/bricks/blob/8fb5e19aa25348980791bda069c89a9647de0c4c/extractors/python_functions/gazetteer/code_snippet.md